src/main/java/browserview/BrowserComponent.java

```diff
@@ -535,40 +567,39 @@ public String getCurrentUrl() {
     }
 
     /**
-     * Switches to the specified tab in GitHub PR page
+     * Opens a specified tab in PR's GitHub page.
+     * Does nothing if specified issue is not a PR.
      *
+     * @param pr TurboIssue of PR whose page is to be opened.
      * @param tabName Either GithubPageElements.DISCUSSION_TAB, GithubPageElements.COMMITS_TAB
      *                or GithubPageElements.FILES_TAB
      */
-    public void switchToTab(String tabName) {
-        if (GitHubURL.isPullRequestLoaded(getCurrentUrl())) {
-            int tabIndex = 0;
-
-            switch (tabName) {
-            case GithubPageElements.DISCUSSION_TAB:
-                tabIndex = 1;
-                break;
-            case GithubPageElements.COMMITS_TAB:
-                tabIndex = 2;
-                break;
-            case GithubPageElements.FILES_TAB:
-                tabIndex = 3;
-                break;
-            default:
-                assert false;
-                return;
-            }
-
-            String xpath = "//*[@id=\"js-repo-pjax-container\"]/div[2]/div[1]/div/div/div[2]/nav/a[%d]";
-            clickElementByXpath(String.format(xpath, tabIndex));
+    public void openPrTab(TurboIssue pr, String tabName) {
+        if (!pr.isPullRequest()) {
+            return;
         }
+        String suffix;
+        switch (tabName) {
+        case GithubPageElements.DISCUSSION_TAB :
+            suffix = GitHubURL.PR_TAB_SUFFIX_DISCUSSION;
+            break;
+        case GithubPageElements.COMMITS_TAB :
+            suffix = GitHubURL.PR_TAB_SUFFIX_COMMITS;
+            break;
+        case GithubPageElements.FILES_TAB :
+            suffix = GitHubURL.PR_TAB_SUFFIX_FILES_CHANGED;
+            break;
+        default:
+            throw new IllegalArgumentException("Argument tabName = " + tabName + " is not a valid PR tab identifier");
```

Comment:
By that logic, assertions shouldn't exist at all :-) Exceptions need catching and handling (more code!). Therefore, exceptions should not be used to handle programmer errors - which are not expected to happen and not expected to be recoverable. 
At the moment we don't have assertions enabled, but we should enable them and keep them enabled indefinitely.